shared/constants/hardware-wallets.ts

```diff
@@ -147,6 +147,14 @@ export const DEVICE_KEYRING_MAP = {
   [HardwareDeviceNames.qr]: KeyringTypes.qr,
 };
 
+export const KEYRING_DEVICE_PROPERTY_MAP = {
```

Comment:
I see a lot of duplication in here. It might be too complex to refactor now but maybe something we can aim to do later on.